mkdocs.yml

```diff
@@ -75,18 +76,18 @@ nav:
       - Blocks gallery: workflows/blocks.md
       - Examples: workflows/gallery_index.md
       - User guide:
-        - Running Workflows: workflows/modes_of_running.md
-        - Workflows Definitions: workflows/definitions.md
-        - Workflow execution: workflows/workflow_execution.md
+          - Running Workflows: workflows/modes_of_running.md
+          - Workflows Definitions: workflows/definitions.md
+          - Workflow execution: workflows/workflow_execution.md
       - Developer guide:
-        - Compiler: workflows/workflows_compiler.md
-        - Execution Engine: workflows/workflows_execution_engine.md
-        - Kinds: workflows/kinds.md
-        - Blocks development: workflows/create_workflow_block.md
-        - Plugins: workflows/blocks_bundling.md
-        - Dynamic Python blocks: workflows/custom_python_code_blocks.md
-        - Versioning: workflows/versioning.md
-        - Testing: workflows/testing.md
+          - Compiler: workflows/workflows_compiler.md
```

Comment:
indent needed to be changed?

Reply:
Prettier auto fixed to maintain the 4-space indent consistently